code/modules/jobs/job_types/detective.dm

```diff
@@ -5,8 +5,8 @@
 	auto_deadmin_role_flags = DEADMIN_POSITION_SECURITY
 	department_head = list(JOB_HEAD_OF_SECURITY)
 	faction = FACTION_STATION
-	total_positions = 1
-	spawn_positions = 1
+	total_positions = 2
+	spawn_positions = 2
```

Comment:
Whereas a change like this, while it technically *could* be in a file under `monkestation/`, is probably better off just being an edit to the original file.

Except, since we made a monkestation-specific change, we'll want to mark it, like so:

```suggestion
	total_positions = 2 //MONKESTATION EDIT: Adding some substance to the detective job
	spawn_positions = 2 //MONKESTATION EDIT: Adding some substance to the detective job
```